runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/state/FlinkStateInternals.java

```diff
@@ -75,7 +80,14 @@
 public class FlinkStateInternals<K> implements StateInternals {
 
   private final KeyedStateBackend<ByteBuffer> flinkStateBackend;
-  private Coder<K> keyCoder;
+  private final Coder<K> keyCoder;
+
+  /**
+   * A set which contains all state descriptors created in the global window. Used for cleanup on
```

Comment:
As implemented this contains all state descriptors, regardless of the window?

Reply:
Yes, that's right but that doesn't matter because all other state should also be cleaned up with the global window.

Reply:
I'll remove "global window".